### ISSUE TITLE
10 sec. rewind (video) (transferred issue)

### ISSUE THREAD

**Issue by Wjxfi** [ismartcoding/plain-app#38]
10 seconds rewind by double-pressing forward to the right of the pause and backward to the left of the pause
https://user-images.githubusercontent.com/113500363/235741365-308be6d5-e36b-4167-a508-e9de6accb96b.mp4

**Comment by WizardOfXerox** [comment]
Yeah... the site of the app will probably need to have a custom web video player for that. currently is uses the default <video> tag in html. so no can do about that until the dev adds it.
